test/integration/node-specific/bson-options/useBigInt64.test.ts

```diff
@@ -0,0 +1,234 @@
+import { expect } from 'chai';
+
+import { BSON, Collection, Db, MongoAPIError, MongoClient } from '../../../mongodb';
+import { setupDatabase } from '../../shared.js';
+
+describe('useBigInt64 option', function () {
+  let configuration;
+  beforeEach(function () {
+    configuration = this.configuration;
+    return setupDatabase(this.configuration);
+  });
+
+  describe('when not provided to client', async function () {
+    let client: MongoClient;
+
+    beforeEach(async function () {
+      client = configuration.newClient(configuration.writeConcernMax());
```

Comment:
Just curious - why are we using `writeConcernMax`? 

Typically, we would either call `this.configuration.newClient()` directly (with no arguments) to get a standard URI, and `this.configuration.newClient({}, options)` to provide options.

Reply:
This was an artefact of copying over the test structure from one of the other bson option tests. I can go ahead and remove these.